extensions/db/src/test/java/org/mule/extension/db/integration/select/SelectMetadataOutputTestCase.java

```diff
@@ -57,6 +57,7 @@ public void selectAll() throws Exception {
     assertFieldOfType(record, "ID", testDatabase.getIdFieldMetaDataType());
     assertFieldOfType(record, "POSITION", testDatabase.getPositionFieldMetaDataType());
     assertFieldOfType(record, "NAME", typeBuilder.stringType().build());
+    //The following assertion will fail in MySQL due that Blob is communicated as a byte[]
```

Comment:
this is not a valid thing to do. Can you use an assumption and skip the assertion if you're using mySQL?